webapp/graphite/render/datalib.py

```diff
@@ -38,6 +39,18 @@ def __init__(self, name, start, end, step, values, consolidate='average'):
     self.options = {}
     self.pathExpression = name
 
+    if tags:
+      self.tags = tags
+    else:
+      self.tags = {'name': name}
+      # parse for tags if a tagdb is configured and name doesn't look like a function-wrapped name
+      if STORE.tagdb and not re.match('^[a-z]+[(].+[)]$', name):
```

Comment:
I think I might be misunderstanding this regex. But shouldn't it also match uppercase chars in function names, like for example `averageSeries(bla)`